dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/settings/FileBasedSettingsFetcherTest.java

```diff
@@ -0,0 +1,157 @@
+package edu.harvard.iq.dataverse.settings;
+
+import com.google.common.io.Files;
+import edu.harvard.iq.dataverse.settings.FileSettingLocations.SettingLocationType;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+import org.junit.jupiter.api.io.TempDir;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+public class FileBasedSettingsFetcherTest {
+
+    private FileBasedSettingsFetcher settingsFetcher;
+    
+    @Test
+    public void loadSettings__PROPERTIES_FROM_CLASSPATH() {
+        
+        // given
+        FileSettingLocations settingLocations = new FileSettingLocations();
+        settingLocations.addLocation(SettingLocationType.CLASSPATH, "/test1.properties", false);
+        
+        FileBasedSettingsFetcher settingsFetcher = new FileBasedSettingsFetcher(settingLocations);
+        
+        // when
+        settingsFetcher.loadSettings();
+        
+        // then
+        assertEquals("keyval1", settingsFetcher.getSetting("BuiltinUsers.KEY"));
+        assertEquals("someval1", settingsFetcher.getSetting(":SomeSetting"));
+        assertEquals("testval1", settingsFetcher.getSetting(":Test1Setting"));
+        assertThat(settingsFetcher.getAllSettings().keySet(),
+                containsInAnyOrder("BuiltinUsers.KEY", ":SomeSetting", ":Test1Setting"));
+    }
+    
+    @Test
+    public void loadSettings__NOT_EXISTING_PROPERTIES_FROM_CLASSPATH() {
+        
+        // given
+        FileSettingLocations settingLocations = new FileSettingLocations();
+        settingLocations.addLocation(SettingLocationType.CLASSPATH, "/notexisting.properties", false);
+        
+        FileBasedSettingsFetcher settingsFetcher = new FileBasedSettingsFetcher(settingLocations);
+        
+        // when
+        Executable loadSettingsOperation = () -> settingsFetcher.loadSettings();
```

Comment:
Isn't `settingsFetcher::loadSettings` more readable?

Reply:
Ok. I will change it